src/reporter.js

```diff
@@ -139,6 +139,10 @@ function createTestRecord(test, specRelativePath) {
   return record;
 }
 
+function normalizePath(pathString) {
+  return pathString.split(path.sep).join('/');
```

Comment:
Should better be doing `replaceAll(path.sep, '/')` - see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll

Also it only should be done if `path.sep !== '/'` (i.e. Windows)